src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CspParameters.cs

```diff
@@ -30,14 +29,14 @@ public CspProviderFlags Flags
             set
             {
                 int allFlags = 0x00FF; // this should change if more values are added to CspProviderFlags
-                Contract.Assert((CspProviderFlags.UseMachineKeyStore |
-                                CspProviderFlags.UseDefaultKeyContainer |
-                                CspProviderFlags.UseNonExportableKey |
-                                CspProviderFlags.UseExistingKey |
-                                CspProviderFlags.UseArchivableKey |
-                                CspProviderFlags.UseUserProtectedKey |
-                                CspProviderFlags.NoPrompt |
-                                CspProviderFlags.CreateEphemeralKey) == (CspProviderFlags)allFlags, "allFlags does not match all CspProviderFlags");
+                Debug.Assert((CspProviderFlags.UseMachineKeyStore |
+                              CspProviderFlags.UseDefaultKeyContainer |
+                              CspProviderFlags.UseNonExportableKey |
+                              CspProviderFlags.UseExistingKey |
+                              CspProviderFlags.UseArchivableKey |
+                              CspProviderFlags.UseUserProtectedKey |
+                              CspProviderFlags.NoPrompt |
+                              CspProviderFlags.CreateEphemeralKey) == (CspProviderFlags)allFlags, "allFlags does not match all CspProviderFlags");
                 //ToDo : Add above error message to resource file
```

Comment:
This TODO can probably be removed now, since Debug.Assert messages don't need to be localized.